### PR TITLE
Update AoM and apa.csl

### PR DESCRIPTION
**Pull request by bwiernik**
- Add support for status variable
- Prefer DOI over other URLs

### CODE REVIEW

File: academy-of-management-review.csl [hunk 1]
```diff
@@ -99,7 +102,14 @@
                 </date>
               </if>
             </choose>
-          </if>
+          </else-if>
+          <else-if variable="issued" match="any">
+            <choose>
```
Comment (adam3smith) [inline academy-of-management-review.csl]: unless I'm missing something, you don't need to test for status here. just use the current line 109.

**Comment by adam3smith** [comment]
thanks -- see the above nit (which is cosmetic, i.e. shouldn't affect output, but I'd like to avoid clutter where possible).

File: academy-of-management-review.csl (hunk 1 above)
Comment (bwiernik) [inline academy-of-management-review.csl]: I think it's necessary. If there is no `issued`, `status` gets printed in its place. `status` shouldn't get printed twice in that case.

**Comment by bwiernik** [comment]
Oh, forgot to change branches when pushing a commit. Also fixed a minor bug in apa.csl

File: academy-of-management-review.csl (hunk 1 above)
Comment (adam3smith) [inline academy-of-management-review.csl]: i understand why you need to test for issued. But why do you need to test for status? You should never need to test for a variable in a `choose` element with just one `if`. That's implicit -- does that make sense? Or what am I missing?

Comment (bwiernik) [inline academy-of-management-review.csl]: Ah, right. I wasn't thinking there.

**Comment by adam3smith** [comment]
thanks!

**Pull request merged**